packages/docs/getting-started/running-a-full-node-in-alfajores.md

```diff
@@ -0,0 +1,127 @@
+# Running a Full Node
+
+This section explains how to get a full node running on the [Alfajores Network](alfajores-testnet.md), using a Docker image that was built for this purpose.
+
+Full nodes play a special purpose in the Celo ecosystem, acting as a bridge between the mobile wallets \(running as light clients\) and the validator nodes. To make sure that full nodes are rewarded for this service, the Celo protocol includes [full node incentives](../celo-codebase/protocol/transactions/full-node-incentives.md).
+
+When a light client sends transactions, they may include a gateway fee to be paid to the node that gossips transactions to the other full nodes and validators.
+
+For this reason, despite the fact that Celo uses a proof-of-stake protocol, users can earn cryptocurrency without first having to own any, simply by running a full node.
+
+{% hint style="warning" %}
```

Comment:
It might be good to stay consistent, but I'm not sure how useful this information is for Alfajores users.

Reply:
We do want people to be aware that this is going to be the case in the future. I agree this would be reworded though. I am going to leave this PR as is simply because I want to get this guide available and the adjustment to this section will need more thought.